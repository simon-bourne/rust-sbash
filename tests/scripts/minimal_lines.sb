pub fn test-fn0() {
} pub fn test-fn1() {
} pub fn test-fn2() {
}
