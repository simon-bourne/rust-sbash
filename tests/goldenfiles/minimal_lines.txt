test-fn0 () { :; }
test-fn1 () { :; }
test-fn2 () { :; }